data/unmatched_slugs.yml

```diff
@@ -0,0 +1,34 @@
+slug_exceptions:
```

Comment:
```suggestion
```

If you remove this and outdent the lines below, you can simplify local_custodian_codes_importer.rb to `@exceptions ||= YAML.load_file("data/unmatched_slugs.yml")`